### DISCUSSION TITLE
Need installation help

### DISCUSSION THREAD

**Discussion by pnson** [openbroadcaster/observer#174] [Q&A]
Hi there,

When I install the Observer, I run the command: tools/cli/ob check. The system reports an error:

OB_SITE (installation web address) does not appear to point to a valid OpenBroadcaster installation.

I am pointing to a specific web server, which is nginx.

![Capture](https://github.com/user-attachments/assets/a54f90e4-f492-4009-8fa5-ae4740cba961)

What should I do to fix this error?

**Comment by firmaillegal** [comment]
What does your config.php file look like?
My example is like this because I configured nginx to run on port 5003
Also if nginx is not configured properly you will have this error.

define('OB_SITE', 'http://localhost:5003/'); // where do you access OB?
